### PR TITLE
Add Ukrainian language

### PR DESCRIPTION
**Pull request by foresle**

### CODE REVIEW

File: _data/trans/ua.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+    "about": "Автори",
+    "contribute": "Форкнути на GitHub",
+    "defaultnote_easy": "Visit the linked page and delete your account.",
+    "defaultnote_email": "Send an email to the provided address and request account deletion.",
```
Comment (tupaschoal) [inline _data/trans/ua.json:5]: Seems like you missed these two.

**Review by tupaschoal — CHANGES_REQUESTED** [review]
Thanks for adding support to a new language! It looks mostly ready, but seems like there are two strings left to translate still, can you please amend so we can publish it?

File: _data/trans/ua.json (hunk 1 above)
Comment (foresle) [reply _data/trans/ua.json]: Really, it's better now

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks a lot for localising JDM into Ukrainian!

**Pull request merged**